src/interpolation.jl

```diff
@@ -147,6 +150,18 @@ function reverse(tinterp::TaylorInterpolant{T,U,N}) where {T<:Real, U<:Number, N
     return TaylorInterpolant(tinterp_rev_t0, tinterp_rev_t, tinterp_rev_x)
 end
 
+@doc raw"""
+    selecteph(eph::TaylorInterpolant{T, U, N}, i::Int) where {T <: Real, U <: Number, N}
+
+Return a `TaylorInterpolant` with only the ephemeris of the `i`-th body. 
+"""
+function selecteph(eph::TaylorInterpolant{T, U, N}, i::Int) where {T <: Real, U <: Number, N}
+    N = numberofbodies(eph)
```

Comment:
Is the `N` in the type-parameters of `eph` the same as the `N` obtained by `numberofbodies(eph)`? If so, I suggest to use the value from the type-parameter; if not, then, use `numberofbodies(eph)`, but perhaps use a different variable name.

Reply:
IIUC, `N` is only used in the call to `nbodyind` to retrieve the indices. If so, then actually the `N` as a type parameter would not be needed, right?

Reply:
Sorry for the typo, the function only works for `TaylorInterpolant{T, U, 2}` and `N` must be the result of `numberofbodies`.